osu.Game/Rulesets/Mods/ModEasy.cs

```diff
@@ -21,7 +22,14 @@ public abstract class ModEasy : Mod, IApplicableToDifficulty, IApplicableFailOve
         public override bool Ranked => true;
         public override Type[] IncompatibleMods => new[] { typeof(ModHardRock) };
 
-        private int retries = 2;
+        [SettingSource("Extra Life count", "Number of extra lives")]
```

Comment:
Thoughts on the option name? Went with "extra life count" there as it sounded better in my head than "Number of allowed retries / attempts"

Reply:
Maybe just "Extra Lives"?